ThirdPartyAdapters/unity/unity/src/main/java/com/google/ads/mediation/unity/UnityAdapter.java

```diff
@@ -171,9 +142,10 @@ public void requestInterstitialAd(Context context,
                                       Bundle mediationExtras) {
         mMediationInterstitialListener = mediationInterstitialListener;
 
-        String gameId = serverParameters.getString(KEY_GAME_ID);
-        mPlacementId = serverParameters.getString(KEY_PLACEMENT_ID);
-        if (!isValidIds(gameId, mPlacementId)) {
+        final String gameId = serverParameters.getString(KEY_GAME_ID);
+        setPlacementId(serverParameters.getString(KEY_PLACEMENT_ID));
```

Comment:
what is added value of `setPlacementId`? can we keep original code `mPlacementId = serverParameters.getString(KEY_PLACEMENT_ID);`?

Reply:
I just thought it was cleaner and more error proof since we had a getPlacementId, but I can put it back

Reply:
let's put back, we can do cleanup as follow up PR. We have already a lot of changes here.